src/commands/install.rs

```diff
@@ -157,12 +162,21 @@ fn archive(version: &Version) -> String {
     format!("ruby-{}.zip", version)
 }
 
-fn build_package(current_dir: &Path, installed_dir: &Path) -> Result<(), FrumError> {
-    debug!("./configure --with-openssl-dir={}", openssl_dir()?);
```

Comment:
As you already know, you can watch this debug log using `RUST_LOG` env like the following.

```
$ RUST_LOG cargo run install 2.6.5
```